test/functional/shared/cli/cli_test.go

```diff
@@ -66,31 +66,37 @@ func verifyRecipeCLI(ctx context.Context, t *testing.T, test shared.RPTest) {
 	// for AWS test and has the AWS scope which the environment created in this does not.
```

Comment:
Unrelated to this pr, but looking into this function overall - Is this comment still valid? Doesn't look the environment is getting switched anywhere in this function.